deploy/docker-compose/.gitignore

```diff
@@ -0,0 +1,2 @@
+.md/
```

Comment:
When I did a "git pull" .env file did not pull down, and neither did all the other folders in this folder. Is this intentional? The "docker-compose" folder only includes the "docker-compose.yaml" after the git pull.